.gitignore

```diff
@@ -3,3 +3,7 @@ env/
 
 __pycache__/
 peewee.db
+
+.idea
+.idea/
```

Comment:
This has nothing to do with the bot! See https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
Thanks i didn't know this.
I'm going to fix it soon.